opencti-platform/opencti-graphql/src/modules/xtm/hub/xtm-hub-client.ts

```diff
@@ -0,0 +1,37 @@
+import { getHttpClient } from '../../../utils/http-client';
+import conf, { logApp } from '../../../config/conf';
+
+type RegistrationStatus = 'active' | 'inactive';
+
+const HUB_BACKEND_URL = conf.get('xtm:hub:backend_url');
+
+export const xtmHubClient = {
+  loadRegistrationStatus: async ({ platformId, token }: { platformId: string, token: string }): Promise<RegistrationStatus> => {
+    const query = `
+      query OctiPlatformEnrollmentStatus($input: OCTIPlatformEnrollmentStatusInput!) {
+        octiPlatformEnrollmentStatus(input: $input) {
+          status
+        }
+      }
+    `;
+
+    const variables = {
+      input: {
+        platformId,
+        token
+      }
+    };
+    const httpClient = getHttpClient({
+      baseURL: HUB_BACKEND_URL,
+      responseType: 'json'
+    });
+
+    try {
+      const response = await httpClient.post('/graphql-api', { query, variables });
+      return response.data.data.octiPlatformEnrollmentStatus.status;
+    } catch (error) {
+      logApp.warn(error);
```

Comment:
```suggestion
      logApp.warn('XTM Hub is not reachable', { reason: error });
```
The warn log is way to much verbose, you need to add a message (I proposed one but feel free to change)and put the error object in metadata.